etc/ci/check_intermittents.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+REPEAT_COUNT=100
+
+for test_type in wpt css; do
+    while read test_name; do
+        echo "  - Checking $test_name"
```

Comment:
Maybe a better message? like, `"Checking $test_name because it's a known intermittent..."`